examples/create-react-app-material-ui/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "create-react-app-material-ui",
```

Comment:
Yes. That would be redundant indeed. It felt right when it was in a separate directory. Will change it to 'create-react-app' only.